rotkehlchen/assets/resolver.py

```diff
@@ -80,7 +80,7 @@ def _get_latest_assets(data_directory: Path) -> Dict[str, Any]:
             return json.loads(remote_asset_data)
 
         # else, same as all error cases use the current one
-    except (RemoteError, KeyError, json.decoder.JSONDecodeError):
+    except (requests.exceptions.ConnectionError, KeyError, json.decoder.JSONDecodeError):
```

Comment:
@kelsos you did not copy my entire diff from discord. I was also removing an import. Niot removing it fails the tests. Run `make lint` before commiting on the backend :smile:

Reply:
No, I planned on fixing it and then things happened give me a moment